rtlamr2mqtt-addon/config.json

```diff
@@ -76,7 +76,8 @@
                 "format": "str?",
                 "unit_of_measurement": "str",
                 "icon": "str",
-                "device_class": "list(none|aqi|battery|carbon_dioxide|carbon_monoxide|current|date|energy|frequency|gas|humidity|illuminance|monetary|nitrogen_dioxide|nitrogen_monoxide|nitrous_oxide|ozone|pm1|pm10|pm25|power_factor|power|pressure|signal_strength|sulphur_dioxide|temperature|timestamp|volatile_organic_compounds|voltag)?"
+                "device_class": "list(none|aqi|battery|carbon_dioxide|carbon_monoxide|current|date|energy|frequency|gas|humidity|illuminance|monetary|nitrogen_dioxide|nitrogen_monoxide|nitrous_oxide|ozone|pm1|pm10|pm25|power_factor|power|pressure|signal_strength|sulphur_dioxide|temperature|timestamp|volatile_organic_compounds|voltag)?",
+                "state_class": "match(^(measurement|total|total_increasing)$)?"
```

Comment:
Any particular reason to use `match` instead of `list`? Wouldn't be better to keep the same pattern? Should we change the others?

Reply:
Oh, I definitely misunderstood what `list` was doing. For some reason I thought it was saying that this attribute could be a list/array of those particular values, but it doesn't look like that's how it works. The schema infers if the attribute can be a list if the attribute actually is a list/array (like `meters`)

In my defense the documentation isn't particularly clear on the matter :)
https://developers.home-assistant.io/docs/add-ons/configuration/#options--schema

I switched it to be `list` instead of `match`

Reply:
No worries! All addon work was made by @AnthonyPluth , I have zero knowledge about it! I am just following his steps! :)